### PR TITLE
[UXE-5745] fix: Edge DNS List View v3 to v4 migration

### PR DESCRIPTION
**Pull request by robsongajunior**
# fix: Edge DNS List View v3 to v4 migration

- Edge App: https://hn93anss06.map.azionedge.net
- Issue: https://aziontech.atlassian.net/jira/software/c/projects/UXE/boards/296?assignee=5b30eea62106432d8401134c&selectedIssue=UXE-5745

### What is the new behavior introduced by this PR?

Only Edge DNS `list and search` **v3** to **v4** API migration.

### Does this PR introduce breaking changes?
- [x] No
- [ ] Yes 

### Does this PR introduce UI changes? Add a video or screenshots here.

- Table List Pagination
- Search with enter searching behavior
- Fix reported by client to no search when the list bigger than 200 items

### Does it have a link on Figma?
- [x] No

<hr />

#### Make sure your pull request fits the checklist below (when applicable):

- [x] The issue title follows the format: [ISSUE_CODE] TYPE: TITLE
- [x] Commits are tagged with the right word (feat, test, refactor, etc)
- [ ] Application responsiveness was tested to different screen sizes
- [x] Code is formatted and linted
- [x] Tags are added to the PR

#### These changes were tested on the following browsers:
- [x] Chrome
- [ ] Edge
- [x] Firefox
- [x] Safari

### CODE REVIEW

File: src/services/edge-dns-services/v4/list-edge-dns-service.js [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+import { makeEdgeDNSBaseUrl } from './make-edge-dns-base-url'
+
+export const listEdgeDNSServiceV4 = async ({
+  fields = '',
+  search = '',
+  ordering = '',
+  page = 1,
+  pageSize = 10
+}) => {
+  const makeListServiceQueryParams = ({ fields, ordering, page, pageSize, search }) => {
```
Comment (robsongajunior) [inline src/services/edge-dns-services/v4/list-edge-dns-service.js:12]: Esta função está neste arquivo devido a API V4 do Edge DNS precisar das aspas ("") no parametro ordering  no GET da lista de Zones.

Após correção na API V4 basta deletar a função e descomentar o import de `import { makeListServiceQueryParams } from '@/helpers/make-list-service-query-params'`

**Review by helio-neto — APPROVED** [review]

File: src/views/EdgeDNS/ListView.vue [hunk 2]
```diff
@@ -152,10 +152,10 @@
         }
       },
       {
-        field: 'status',
+        field: 'active',
         header: 'Status',
         type: 'component',
-        filterPath: 'status.content',
+        filterPath: 'active.content',
```
Comment (pauloSF0) [inline src/views/EdgeDNS/ListView.vue:158]: deve adicionar isso aqui: sortField: 'active',

Comment (robsongajunior) [reply src/views/EdgeDNS/ListView.vue:158]: apenas adicionar ou remover o filePath?

Comment (pauloSF0) [reply src/views/EdgeDNS/ListView.vue:158]: remover esse filterPath e adicionar sortField

**Review by HerbertJulio — APPROVED** [review]

**Pull request merged**